client/src/App.js

```diff
@@ -1,13 +1,21 @@
-import React, { useEffect } from 'react';
-import axios from 'axios';
-import Post from './post-component/index';
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Login from './login-component/index';
 import Signup from './signup-component/index';
+import Feed from './feed-component/index';
 // global sass - contains theming
 import './App.global.scss';
 
 const App = () => {
-  return <Signup></Signup>;
+  return (
+    <Router>
+      <Switch>
+        <Route exact path='/signup' component={Signup} />
+        <Route exact path='/login' component={Login} />
+        <Route exact path='/' component={Feed} />
```

Comment:
looks good but i feel like we should reserve '/' for initial landing page (when users aren't signed in or on a log in page). path='/home' makes more sense to me at the moment